l10n_it_fatturapa_in/models/account.py

```diff
@@ -200,9 +206,11 @@ def set_einvoice_data(self, fattura):
             fattura.DatiBeniServizi.DatiRiepilogo)
         amount_tax = self.compute_xml_amount_tax(
             fattura.DatiBeniServizi.DatiRiepilogo)
+        amount_rounding = self.compute_xml_rounding_total(
+            fattura.DatiBeniServizi.DatiRiepilogo)
```

Comment:
Gli arrotondamenti ci sono anche su ``FatturaElettronicaBody.DatiGenerali.DatiGeneraliDocumento``

Reply:
il campo sì, però quel :eagle: che ha emesso la fattura non l'ha messo

Reply:
mi riferisco al programmatore ovviamente, non all'utente, comunque è un caso strano perchè l'arrotondamento c'è solo nelle righe e nel dettaglio, non nel totale in pagamento